src/analyze/analyze-verify.c

```diff
@@ -141,9 +141,10 @@ static int verify_executables(Unit *u) {
 
         assert(u);
 
-        exec =  u->type == UNIT_SOCKET ? SOCKET(u)->control_command :
+        exec = u->type == UNIT_SOCKET ?
+                SOCKET(u)->control_command :
                 u->type == UNIT_MOUNT ? MOUNT(u)->control_command :
-                u->type == UNIT_SWAP ? SWAP(u)->control_command : NULL;
+                                        u->type == UNIT_SWAP ? SWAP(u)->control_command : NULL;
```

Comment:
Hmm... I do not like this indentation...

Reply:
Matching `?` and `:` are aligned. I think it's a reasonable change.

Reply:
??? Sorry, I do not understand... Which `?` and `:` are aligned?

Reply:
Github is displaying this wrong, and makes it seem like it's unaligned.
The second and third argument to the ternary operator are aligned to the same column.
This is different than what is done in various other places, but it is quite reasonable.
But if you don't like it, I can drop it.

Reply:
> Github is displaying this wrong,

Ah, I see.

I am not sure how we should indent expressions with ternary operators. But, at least in this case, I prefer the original indentation.

Reply:
Dropped.